packages/celotool/src/lib/geth.ts

```diff
@@ -468,126 +467,168 @@ const measureBlockscout = async (
   }
 }
 
+export const transferCeloGold = async (
```

Comment:
I was never quite a fan of putting all the code for the light client here in this massive file, maybe consider separating it